modvendor/modvendor_test.go

```diff
@@ -111,31 +948,28 @@ func TestModVendorWithRef(t *testing.T) {
 	// or else the test passes even if the correct ref is not used.
 	repogit.Checkout(ref)
 
-	newCloneDir, err := modvendor.Vendor(rootdir, vendordir, tf.Source{
+	source = tf.Source{
```

Comment:
hmm here we kept the tf.Source initialization instead of parsing the URL :thinking: (Im probably missing something on the other ones)

Reply:
I just prefer the other way, as I explained, it could last longer as we use only the Parse API.

Reply:
fair enough, but then we need to fix it here to use the parse no ?

Reply:
it depends on if you agree or not... I can revert the code to the way it was before.

Reply:
lets use the parse function :+1: